bootstrap/src/main/java/com/navercorp/pinpoint/bootstrap/config/ProfilerConfig.java

```diff
@@ -140,6 +140,8 @@ public static ProfilerConfig load(String pinpointConfigFileName) throws IOExcept
     private boolean memcached = true;
     private boolean memcachedKeyTrace = false;
     
+    private Filter<String> jettyExcludeUrlFilter = new SkipFilter<String>();
```

Comment:
No, it's not using ProfilerConfig.jettyExcludeUrlFilter. Among members of ProfilerConfig, JettyConfiguration only uses readString(), which is not accessing jettyExcludedUrlFilter.